api/v1/orchestratorscan/service.go

```diff
@@ -122,7 +122,7 @@ func (s *serviceImpl) GetIstioVulnerabilities(_ context.Context, req *v1.GetIsti
 	version, err := convert.TruncateVersion(req.GetIstioVersion())
```

Comment:
nit: `v` instead of `version`, as `version` is also the name of a package this file requires.